Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -453,6 +454,16 @@ auto RenderLayerCompositor::BackingSharingState::backingProviderCandidateForLaye
     return &candidate;
 }
 
+auto RenderLayerCompositor::BackingSharingState::previousBackingProviderCandidateForLayer(const RenderLayer& layer) -> Provider*
```

Comment:
Should this be `existing` not `previous`? I'm not sure what sense of "previous" is intended; it's not previous in traversal order.

Reply:
I meant 'previous' rendering update, but maybe that's not commonly used terminology here. I can change to `existing` instead.